### PR TITLE
Feature/grafana local dashboards

### PR DESCRIPTION
**Pull request by SomeFriendlyGhost**
- Added config map
- Added local dashboards as .JSON files
- Removed content from values file
- Version bump

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -1,5 +1 @@
-grafana:
-  dashboards: 
-    default:
-      custom-dashboard:
-        file: dashboards/custom-dashboard.json
+#this is a values file
```
Comment (klausenbusk) [inline chart/values.yaml:1]: Do we need a empty values file?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 Umbrella chart for Grafana
 
 ## Adding a dashboard
+### Overview
 To add a Grafana dashboard to grafana, you will need to:
 1. Add the .JSON file to the dashboards folder
-2. Add the dashboard in the values.yaml file so it will be imported by Grafana
+
+### Step 1
```
Comment (klausenbusk) [inline README.md:9]: Should we remove line 5 to 9? It feels a bit redundant.

Comment (SomeFriendlyGhost) [reply README.md:9]: good point there is no reason for an overview when there is only 1 step.

File: chart/values.yaml (hunk 1 above)
Comment (SomeFriendlyGhost) [reply chart/values.yaml:1]: nope we dont need it so it can be removed

File: README.md [hunk 3]
```diff
@@ -2,6 +2,6 @@
 Umbrella chart for Grafana
 
 ## Adding a dashboard
-To add a Grafana dashboard to grafana, you will need to:
-1. Add the .JSON file to the dashboards folder
-2. Add the dashboard in the values.yaml file so it will be imported by Grafana
+In the service-grafana repository dashboards can be stored locally. To do so, find the link to the dashboard and download the .JSON file to the service-grafana/chart/dashboards folder.
+Remember to rename the dashboard so it follows the same standard as the others.
```
Comment (skovbakke) [inline README.md:6]: If we have a certain standard, I think we should say explicitly what it is.

**Review by skovbakke — APPROVED** [review]
Only a single comment. Otherwise it looks good.

File: README.md (hunk 3 above)
Comment (SomeFriendlyGhost) [reply README.md]: The comment has a valid point and the wording has been changed

File: chart/templates/dashboards.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{{- $root := .}}
```
Comment (klausenbusk) [inline chart/templates/dashboards.yaml:1]: Why do we need this?

Comment (jaxels10) [reply chart/templates/dashboards.yaml:1]: True, we don't need that. In general it is used in helm to keep the root scope saved in a variable, but this is not used here.

Comment (skovbakke) [reply chart/templates/dashboards.yaml:1]: Jesper, does the scope not change when you use `range`?

**Comment by klausenbusk** [comment]
Please squash when https://github.com/distributed-technologies/services-grafana/pull/6#discussion_r713694169 has been sorted and it LGTM 🎉

File: chart/templates/dashboards.yaml (hunk 4 above)
Comment (skovbakke) [reply chart/templates/dashboards.yaml:1]: @jaxels10

Comment (jaxels10) [reply chart/templates/dashboards.yaml:1]: Ahh I missed that we used the variable inside the range loop. Yes, the scope changes with the range loop and .Files.Get should be called from the root dir, therefore this variable should stay the way it is. An alternative approach would be to declare a $files := .Files and thereby your .Files object will always have the scope of the root dir

**Review by jaxels10 — APPROVED** [review]
I think this looks good now

**Pull request merged**